INSTALL.md

```diff
@@ -5,6 +5,8 @@ System requirements: GalSim currently only supports Linux and Mac OSX.
 
 Table of Contents:
 
+0. [Overall summary](#0-overall-summary)
```

Comment:
Mmm.  This is a classic markdown [bug](http://stackoverflow.com/questions/15078393/begin-ordered-list-from-0-in-markdown)-that-they-think-is-a-[feature](http://daringfireball.net/projects/markdown/syntax#list).  They finally fixed this on [StackOverflow](http://meta.stackexchange.com/questions/137077/cant-start-a-numbered-list-on-a-number-other-than-1), but [GitHub still forces you](https://docs.gitlab.com/ee/user/markdown.html#lists) to start lists at 1.

The workarounds that I know about are
0. put a `\` before the dot: `0\.`, etc. 
0. use a right paren instead: `0)`.  Or `(0)` perhaps.
0. renumber everything starting with 1.
0. use [custom html code](http://stackoverflow.com/a/15078486/1332281)